### DISCUSSION TITLE
Create data for contentType

### DISCUSSION THREAD

**Discussion by datdv260497** [OrchardCMS/OrchardCore#8679] [Q&A]
I have problem when insert data because Content Type contain Time Field.
I get error :
Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint "FK_ContentItemIndex". The conflict occurred in database "Orchard", table "dbo.Document", column 'Id'.
Arithmetic overflow error converting expression to data type datetime.
What can i do to solve this error ?
Thank !

**Comment by deanmarcussen** [comment]
What version are you running on? There were some date time issues a little while back, but they should be resolved on the latest dev.

**Comment by TFleury** [comment]
It's related to Content Fields Indexing (SQL) feature
I opened an issue : #9070
